crawl-ref/source/dat/des/branches/shoals.des

```diff
@@ -2110,6 +2110,53 @@ cww......L
 ccccJJKKLL
 ENDMAP
 
+NAME:   dreamdust_from_the_deep
+TAGS:   no_monster_gen no_item_gen no_tide
+DEPTH:  Shoals, !Shoals:$
+ORIENT: northwest
+WEIGHT: 3
+KFEAT:  X = open_sea
```

Comment:
Avoid using glyphs that already have a default definition as described in the DES syntax doc. You will see older vaults use `X` quite a bit in this way, but this is something we discourage. It works much better for syntax highlighting and vault reading in general if you can see a standard glyph and know that it has its usual definition. I recommend using a simple scheme for custom glyph usage that you try to follow consistently. For example, try using the lower-case glyphs `apqrsyz` for custom solid terrain elementals (skipping the item list glyphs on `defghijk`). You can use `DEFHJKLMNOPQRSZ` either alternately if you prefer, or specifically for things that will be eventually some kind of open terrain. For custom loot items, it's recommended to use `defghijk` even if definiting via `KITEM` and likewise it's recommended to use `1234567` with either `MONS` or `KMONS` whenever possible. If you run out of numeric glyphs, you can fall back to e.g. upper-case glyphs mentioned above. 

Generally avoid using crawl display glyphs like `z` or `Z` or `#` just because those are crawl console glyphs when numeric or other appropriate glyphs are available. The glyphs ``-'"`;!`` are great to use for large areas when placing monsters or to designate inside/outside areas for vault logic.